MovieApp/Scenes/Home/CollectionViewCell/MoviesCollectionViewCell.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  MoviesCollectionViewCell.swift
+//  MovieApp
+//
+//  Created by tornike ivanashvili on 17.07.23.
+//
+
+import UIKit
+
+final class MoviesCollectionViewCell: UICollectionViewCell {
+    
+    private let moviePoster: UIImageView = {
+        let image = UIImageView()
+        image.contentMode = .scaleAspectFill
+        image.translatesAutoresizingMaskIntoConstraints = false
+        return image
+    }()
+    
+    private let favouriteButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: Constants.FavoriteButton.image), for: .normal)
```

Comment:
აქ  სელექთის ლოგიკა გაქ დასამაატებელი. button.setImage(UIImage(named: Constants.FavoriteButton.selectedImage), for: .selected) da button.setImage(UIImage(named: Constants.FavoriteButton.unselected), for: .normal) toggle() - ით გააკეთე.